tests/shared/hardware_fonduer_model.py

```diff
@@ -0,0 +1,54 @@
+import pickle
```

Comment:
Should this file in the packaging folder or here? It seems like it's specific to the packaging test?

Reply:
This should be here because `HardwareFonduerModel._classify` uses it to unpickle "parts_by_doc_dict.pkl", which is specific to `HardwareFonduerModel`.